Gems/Compression/Platform/Android/android_gem.cmake

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) Contributors to the Open 3D Engine Project.
```

Comment:
You are removing these platform specific gem.json files right? According to: https://github.com/o3de/o3de/pull/9606

Reply:
Yes, the platform specific gem.json files are removed, but not the platform specific cmake files.

Reply:
Actually I'll update the gem template to remove these files.